frontend/public/components/utils/resource-link.tsx

```diff
@@ -68,49 +72,118 @@ export const resourcePath = (kind: K8sResourceKindReference, name?: string, name
 export const resourceObjPath = (obj: K8sResourceKind, kind: K8sResourceKindReference) =>
   resourcePath(kind, _.get(obj, 'metadata.name'), _.get(obj, 'metadata.namespace'));
 
-export const ResourceLink = connectToModel(
-  ({
-    className,
-    displayName,
-    inline = false,
-    kind,
-    linkTo = true,
-    name,
-    namespace,
-    hideIcon,
-    title,
-    children,
-  }) => {
-    if (!kind) {
-      return null;
+const resourceLinkStateToProps = (state: RootState): resourceLinkStateProps => {
+  return {
+    activeNamespace: state.UI.get('activeNamespace'),
+    receivedCSVs: state.k8s.getIn(['CSVS', 'csvs']),
+  };
+};
+
+export const compareKinds = (csv, owner) => {
```

Comment:
This isn't really comparing kinds. It's checking the CSV against an owner ref. Maybe call this `isOwnedByOperator`.

Please add types to the arguments.